crates/astria-sequencer/src/proposal/commitment.rs

```diff
@@ -30,6 +30,25 @@ impl GeneratedCommitments {
     }
 }
 
+pub(crate) trait Example {
+    fn example() -> Self;
+    fn commitments_size(self) -> usize;
+}
+
+impl Example for GeneratedCommitments {
+    fn example() -> Self {
+        GeneratedCommitments {
+            rollup_datas_root: [0; 32],
+            rollup_ids_root: [0; 32],
+        }
+    }
+
+    // used for accounting for commitments' data size in cometbft block
+    fn commitments_size(self) -> usize {
```

Comment:
I'm not sure what the idiomatic way to get the size of a struct's inner parts before having a concrete version of the struct is. I didn't want to hardcode the size in case changes were made later to the `GeneratedCommitments` logic/struct.

Reply:
left a comment, you can use `std::mem::size_of`